pallets/pallet-bonded-coins/src/curves.rs

```diff
@@ -0,0 +1,253 @@
+use core::ops::{AddAssign, BitOrAssign, ShlAssign};
+use parity_scale_codec::{Decode, Encode, MaxEncodedLen};
+use scale_info::TypeInfo;
+use sp_arithmetic::ArithmeticError;
+use substrate_fixed::{
+	traits::{Fixed, FixedSigned, ToFixed},
+	transcendental::{exp, ln, sqrt},
+	types::I9F23,
+};
+
+#[derive(Clone, Debug, Encode, Decode, PartialEq, Eq, TypeInfo, MaxEncodedLen)]
+pub enum Curve<F> {
```

Comment:
[0a3481b](https://github.com/KILTprotocol/kilt-node/pull/765/commits/0a3481bcca9c45104e357f59f50ca6a36f962492)

Reply:
Isn't this more like `Coefficients`? Or something like that? Correct me if I'm wrong. `Parameter` is still not explicative enough, sorry.

Reply:
"In the context of the Curve, it represents a coefficient. 

For calculate_costs, the inputs active_issuance_pre, active_issuance_post, and op represent balances. 

It will never be fully declarative. Introducing another generic type for calculate_costs just for better naming isn't worth from my pov

Reply:
Ok, then I'll resolve this comment 😀